app/models/user.rb

```diff
@@ -32,6 +34,9 @@
 #
 
 class User < ApplicationRecord
+  CELLS = %i[marketing ops presidential publicity technical training welfare]
```

Comment:
Don't think need ops here

Reply:
Oh, cos in current duty website have "Center and Ops". So just remove?